File: JAVA/APE6_2.java
package APE6;

import java.util.ArrayList;

public class APE6_2 {
    public static void main(String[] args) {
        ArrayList<Double> notas = new ArrayList<>();

        //Agregar elementos al ArrayList
        notas.add(9.8);
        notas.add(8.75);
        notas.add(7.5);
        notas.add(10.0);
        notas.add(5.3);
        notas.add(9.2);

        //Mostrar elementos
        for(int i=0; i<notas.size(); i++){
            System.out.println("Nota " + (i+1) + ": " + notas.get(i));
        }
    }
}

File: C++/APE6_2.cpp
#include <iostream>
#include <vector>
using namespace std;
 
int main() {
    vector<double> notas;
 
    // Agregar 6 notas al vector
    notas.push_back(8.5);
    notas.push_back(7.0);
    notas.push_back(9.2);
    notas.push_back(6.8);
    notas.push_back(10.0);
    notas.push_back(5.5);
 
    // Mostrar todas las notas
    cout << "Lista de notas:" << endl;
    for (int i = 0; i < notas.size(); i++) {
        cout << "Nota " << (i + 1) << ": " << notas[i] << endl;
    }

    return 0;
}